docs/CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -0,0 +1,167 @@
+
+# v1.10.0
+## Downloads for v1.10.0
+
+Download v1.10.0 in the [v1.10.0 release page](https://github.com/karmada-io/karmada/releases/tag/v1.10.0).
+
+## What's New
+
+### Workload Rebalance
+
+This release introduced a workload rebalancing capability. It can actively trigger a brand fresh rescheduling to establish an entirely new replica distribution across clusters. With the workload rebalancing capability, users can trigger a workload rebalancing on demand if the current replicas distribution is not optimal.
+
+See [Proposal of introducing a rebalance mechanism to actively trigger rescheduling of resource](https://github.com/karmada-io/karmada/blob/master/docs/proposals/scheduling/workload-rebalancer/workload-rebalancer.md) for more details.
+
+(Feature contributors: @chaosi-zju)
+
+### Free the resource template name from the limitation of label length
+
+Previously, since the resource template name was eventually assembled as part of the label `work.karmada.io/name`, its length will be limited to less than 64. Starting from `v1.8.0`, Karmada has introduced label `permanent-id` to replace the role of resource template names in labels through smooth migration. 
+
+In this release, a lot of effort was spent to remove the deprecated labels and make label `permanent-id` work, freeing the length of the resource template name from the limitation of the label length.
+
+See [[Umbrella] Use permanent-id to replace namespace/name labels in the resource](https://github.com/karmada-io/karmada/issues/4711) for more details.
+
+(Feature contributors: @liangyuanpeng, @whitewindmills, @XiShanYongYe-Chang)
+
+### Usability and reliability Enhancements in the production environment
+
+In this release, Karmada has received a great deal of feedback from its users, which has helped us make a number of enhancements, invoking:
+
+- Capability Enhancement: supporting propagating `Secret` of type `kubernetes.io/service-account-token`, optimizing deprioritized policy preemption logic, significantly reducing memory usage of `karmada metrics adapter`, karmada resource interpreter allows for more pruners to be added and avoid lint errors such as `cyclomatic complexityand`, syncing deployment replicas when it is controlled by `HorizontalPodAutoscaler`, and so on.
+
+- Security Enhancements: upgrading rsa key size from 2018 to 3072, granting the correct permissions when creating files, some necessary measures to silence false positives from security scanning tools, like adjusting log print content when delete token by karmadactl and silence gosec warning G107, and so on.
+
+- Software Ecosystem: showing `status.labelSelector` for `CloneSet`, integrated with OIDC authentication for cluster operation auditing and access control, and so on.
+
+We believe these efforts will make Karmada even better!
+
+(Feature contributors: @a7i, @chaosi-zju, @chaunceyjiang, @guozheng-shen, @veophi, @whitewindmills, @zhzhuang-zju)
```

Comment:
Let's take this part TBD.
Because there are still a lot of feature enhancements and bugfixes based on production practice. We can't list them all here.